packages/openbox.rb

```diff
@@ -33,25 +34,87 @@ def self.build                   # the steps required to build the package
 
   def self.install                 # the steps required to install the package
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "sudo ld /bin/bash /bin/sh"
+    # For Chrome OS/Fyde OS with android arc container support
+    # Use XServer XSDL
     system "echo '#!/bin/bash' > startopenbox"
-    system "echo 'stopsommelier' >> startopenbox"
-    system "echo 'echo am start x.org.server/.MainActivity | sudo android-sh' >> startopenbox"
     system "echo 'echo Starting XServer XSDL' >> startopenbox"
+    system "echo 'echo am start x.org.server/.MainActivity | sudo android-sh' >> startopenbox"
+    
+    ## Remove the comment if the xorg_xdpyinfo and 'adwaita_icon_theme' package is ready
+    # system "cat <EOF>> startopenbox
+    ## Wait for XSDL X server
+    #MAX_ATTEMPTS=120 # About 60 seconds
+    #COUNT=0
+    #stopsommelier
+    #echo -n \"Waiting for X server to be ready...\"
+    #while ! xdpyinfo -display ${DISPLAY} >/dev/null 2>&1; do
+    #  echo -n \".\"
+    #  sleep 0.50s
+    #  export DISPLAY=100.115.92.2:0
+    #  COUNT=$(( COUNT + 1 ))
+    #  if [ \"${COUNT}\" -ge \"${MAX_ATTEMPTS}\" ]; then
+    #    echo \"  Gave up waiting for X server on ${DISPLAY}\"
+    #    exit 1
+    #  fi
+    #done
+    #echo \"  Done - X server is ready!\"
+    #EOF"
     system "echo 'sleep 7' >> startopenbox"
     system "echo 'export DISPLAY=100.115.92.2:0' >> startopenbox"
-    system "echo 'openbox' >> startopenbox"
-    system "echo 'echo XServer closed or logged out session' >> startopenbox"
+    system "echo 'openbox \"$@\"' >> startopenbox"
     system "echo 'startsommelier' >> startopenbox"
-    system "echo 'export DISPLAY=:0' >> startopenbox"
-    system "install -Dm755 startopenbox #{CREW_DEST_PREFIX}/bin/startopenbox"
+    # For Chrome OS/Chromium OS/Cloudready without android arc container support
+    # Use 'xinit'
+    system "if [[ cat /etc/lsb-release | grep ARC != "" ]]; then echo 'Your OS have Android Support :)'; else if [[ whereis xinit | grep /usr/local/bin/xinit != /usr/local/bin/xinit ]]; then echo 'To use this package, You need to install `xinit` package'; else echo 'pkill twm && openbox &' >> /usr/local/etc/X11/xinit/xinitrc; fi"
```

Comment:
Can this line be split into separate lines for clarity?  Also, `ARC != ""` should be `ARCH != \"\"`?  Finally, please abstract the directories and change `/usr/local` to `#{CREW_PREFIX}`.

Reply:
the `ARC` is used to detect does the system have android container support, maybe change to `grep CHROMEOS_ARC_ANDROID_SDK_VERSION` will be more clearly